docs/GraFx-Studio/guides/template-variables/date/index.md

```diff
@@ -0,0 +1,110 @@
+# Date Variables
+
+## Create a Date Variable
+
+Under the Automate icon, click Variables, and add a variable with the "+" sign.
+
+![screenshot](date0.png)
+
+Choose "Date" as the Variable Type.
+
+## Set the Variable Name
+
+Double-click the name, or choose "Rename" under the "..." menu.
+
+![screenshot](date1.png)
+
+## Set General Properties
+
+![screenshot-full](date2.png)
+
+### Visible
+
+Defines if the variable will be visible in the [Studio UI](/GraFx-Studio/concepts/template-management/#studio-ui) to the end user.
+
+### Display Format
+
+A date can be displayed in several formats, depending on the local preferences. You can set a formatting string, how to display the date.
+
+The formatting can be empty, and will then default to "dd/MM/yyyy"
+
+**Format**
+
+Supported date formats
+
+- Day -> `d`, `dd`
+- Month -> `M`, `MM`, `MMM`, `MMMM`
+- Year -> `yy`, `yyyy`
+- Day of week -> `ccc`, `cccc`
+
+[See Github](https://unicode-org.github.io/icu/userguide/format_parse/datetime/) for ICO formatting. (Only the above formats are supported)
+
+Examples for an input date of `10-12-1815`
+
+- Format `dd/MM/yyyy` will display `12/10/1815`
+- Format `d.M.yy` will display `12.10.15`
```

Comment:
Using 10-12-1815 as an example does not show the difference for days and months between the first 2 example formats. It would be clearer if you would use a day and month with a single digit, like the example in the workspace (April 1, 2030).

Reply:
check, Monday April 1st it is, 1815

Reply:
Now I'm curious about 1815. Is there a story behind that?